### PR TITLE
WIP: update contexts for new section name and unique naming conventions

### PR DESCRIPTION
**Pull request by michelle-luna**

### CODE REVIEW

**Comment by KunalJain** [comment]
Lgtm. @ataggart do we need to call out anything else?

File: jekyll/_cci2/contexts.md [hunk 1]
```diff
@@ -21,13 +21,15 @@ Contexts are created on the Settings page of the CircleCI application, in the Or
 
 To use environment variables set on the Contexts page, the person running the workflow must be a member of the organization for which the context is set and the rule must allow access to all projects in the org. The default context name is `org-global` and the default rule allows access to all projects in the org.
 
+Context names must be unique for each Github or Bitbucket organization. For CircleCI installations on your own server or private cloud, Context names must be unique across the installation.
+
 ## Using a Context
 
-1. Navigate to the Settings > Contexts page in the CircleCI application. The Create a Context button appears.
+1. Navigate to the Settings > Contexts page in the CircleCI application. The Create a Context button appears if you have no existing contexts.
```
Comment (AlexanderMann) [inline jekyll/_cci2/contexts.md]: I think in the post multi contexts world, the `create context` button _always_ exists

Comment (AlexanderMann) [reply jekyll/_cci2/contexts.md]: It just depends on where on the screen it is (ie, in the empty state or in the header)

File: jekyll/_cci2/contexts.md [hunk 2]
```diff
@@ -21,13 +21,15 @@ Contexts are created on the Settings page of the CircleCI application, in the Or
...
 
-2. Click the Create Contexts button. The default name and rules appear with an Add a Resource button.
+2. Click the Create Contexts button. The default name and rules appear with an Add Environment Variable button.
 
-3. Click the Add a Resource button and fill in the variable name and value. Click the Add Variable button to save it.
+3. Click the Add Environment Variable button and type in the variable name and value. Click the Add Variable button to save it.
```
Comment (AlexanderMann) [inline jekyll/_cci2/contexts.md]: I'm not sure if `type` is misleading here as you can copy paste that value as well (which is actually more recommended for sensitive values)

File: jekyll/_cci2/contexts.md [hunk 3]
```diff
@@ -40,9 +42,19 @@ workflows:
             context: org-global
 ```
 
-To rerun a job and use the context, it must be rerun from the Workflows page of the CircleCI application. Jobs invoked using the Rebuild button on the Builds page will **not** use the context defined in the workflow. 
+To rerun a job and use the context, it must be rerun from the Workflows page of the CircleCI application.
```
Comment (AlexanderMann) [inline jekyll/_cci2/contexts.md]: might want to highlight `must` here, this is something which trips people up all the time

File: jekyll/_cci2/contexts.md [hunk 4]
```diff
@@ -21,13 +21,15 @@ Contexts are created on the Settings page of the CircleCI application, in the Or
 
 To use environment variables set on the Contexts page, the person running the workflow must be a member of the organization for which the context is set and the rule must allow access to all projects in the org. The default context name is `org-global` and the default rule allows access to all projects in the org.
 
+Context names must be unique for each Github or Bitbucket organization. For CircleCI installations on your own server or private cloud, Context names must be unique across the installation.
```
Comment (ataggart) [inline jekyll/_cci2/contexts.md]: @AlexanderMann The second sentence above is incorrect, right?

Comment (AlexanderMann) [reply jekyll/_cci2/contexts.md]: I don't _think_ so...can't Server installations only have a _single_ Organization? It'd have the same effect...

File: jekyll/_cci2/contexts.md (hunk 1 above)
Comment (michelle-luna) [reply jekyll/_cci2/contexts.md]: oh right, thanks, I removed that sentence.

File: jekyll/_cci2/contexts.md (hunk 2 above)
Comment (michelle-luna) [reply jekyll/_cci2/contexts.md]: thanks, changed to copy/paste

File: jekyll/_cci2/contexts.md (hunk 3 above)
Comment (michelle-luna) [reply jekyll/_cci2/contexts.md]: thanks, added

File: jekyll/_cci2/contexts.md (hunk 4 above)
Comment (michelle-luna) [reply jekyll/_cci2/contexts.md]: @smaant maybe you can review this as well, please

**Comment by KunalJain** [comment]
🚢 🇮🇹

**Pull request merged**